### PR TITLE
Improved contentful-persistence.swift compatibility

### PR DESCRIPTION
**Pull request by loudmouth**

### CODE REVIEW

File: Sources/Client.swift [hunk 1]
```diff
@@ -253,6 +272,45 @@ open class Client {
...
+    /**
+     Fetch the space this client is constrained to.
+
+     - Parameter completion: A handler being called on completion of the request.
+
+     - Returns: The data task being used, which enables cancellation of requests, or `nil` if the.
+     Space was already cached locally
+     */
+    @discardableResult public func fetchSpace(then completion: @escaping ResultsHandler<Space>) -> URLSessionDataTask? {
+        // Attempt to pull from cache first.s
```
Comment (mariobodemann) [inline Sources/Client.swift]: `first.s`? I think there might be something missing ... ;)

File: Sources/ClientConfiguration.swift [hunk 2]
```diff
@@ -10,7 +10,7 @@ import Foundation
 
 enum Defaults {
     static let cdaHost = "cdn.contentful.com"
-    static let locale = "en-US"
+//    static let locale = "en-US"
```
Comment (mariobodemann) [inline Sources/ClientConfiguration.swift]: 😢

Comment (loudmouth) [inline Sources/ClientConfiguration.swift]: yeah reverting that...i noticed it wasn't used. should probably just be deleted.

Comment (mariobodemann) [inline Sources/ClientConfiguration.swift]: 👍

File: Sources/Client.swift (hunk 1 above)
Comment (loudmouth) [inline Sources/Client.swift]: 👍

**Pull request merged**